mmdet/core/anchor/point_generator.py

```diff
@@ -0,0 +1,33 @@
+import torch
+
+class PointGenerator(object):
+
+    def _meshgrid(self, x, y, row_major=True):
```

Comment:
hi, kai, thanks for your comment. In fact, I follow your _meshgrid function in anchor_generator.

Reply:
Yes we implemented a `_meshgrid()` because there was no `torch.meshgrid()` at that time. It will also be replaced with the build-in methods of Pytorch.